include/WCSimWCHit.hh

```diff
@@ -157,6 +160,7 @@ class WCSimWCHit : public G4VHit
   std::vector<G4ThreeVector> photonEndPos;
   std::vector<G4ThreeVector> photonStartDir;
   std::vector<G4ThreeVector> photonEndDir;
+  std::vector<std::string>   photonCreatorProcess;
```

Comment:
```suggestion
  std::vector<ProcessType_t>   photonCreatorProcess;
```
This'll fix the compliation error that is currently stopping the CI build. You may need to make similar changes in other files